### ISSUE TITLE
crash when disabling account

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dequis/msn-pecan#171]
```
What steps will reproduce the problem?
1.disable the account in pidgin

What version of the product are you using? On what operating system?
0.0.19 
pidgin 2.5.5 if that matters (i doubt it tho)


nothing more that i can say lol
```

Original issue reported on code.google.com by `dac...@gmail.com` on 28 Jun 2009 at 10:11

**Comment by GoogleCodeExporter** [comment]
```
What operating system? Windows?
```

Original comment by `felipe.contreras` on 29 Jun 2009 at 1:11
- Added labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
ye srry, im using windows XP SP3
```

Original comment by `dac...@gmail.com` on 29 Jun 2009 at 2:50

**Comment by GoogleCodeExporter** [comment]
```
There's a fix when logging off in git, will be available on 0.0.20, please 
verify one
it's released.
```

Original comment by `felipe.contreras` on 6 Jul 2009 at 12:41
- Changed state: **Fixed**
- Added labels: **Priority-Medium**, **Milestone-0.1.0**
